src/components/KTabs/KTabs.vue

```diff
@@ -18,9 +18,9 @@
         @keydown.enter.prevent="handleTabChange(tab.hash)"
         @keydown.space.prevent="handleTabChange(tab.hash)"
       >
-        <a class="tab-link">
+        <span class="tab-link">
```

Comment:
If we're already going to change the structure, I'm wondering if we should make this a `div` rather than a `span` to allow semantic HTML based on the notion that a user could literally slot any HTML element.

Reply:
Changed in https://github.com/Kong/kongponents/pull/1250/commits/abbeb7e26993380252edd1686e999d8e2a46b3ab.